server/service/integration_live_queries_test.go

```diff
@@ -215,9 +216,7 @@ func (s *liveQueriesTestSuite) TestLiveQueriesRestOneHostOneQuery() {
 				hostDistributedQueryPrefix + cid:    0,
 				hostDistributedQueryPrefix + "9999": "0",
 			},
-			Messages: map[string]string{
-				hostDistributedQueryPrefix + cid: "some msg",
-			},
+			Messages: map[string]string{},
```

Comment:
Why this change? Was it showing up as error?

Reply:
Yes.